packages/react/src/utils/classNames.ts

```diff
@@ -1 +1,3 @@
-export default (...args) => args.filter((e) => e).join(' ');
+import classnames from 'classnames/dedupe';
```

Comment:
Oh this seems like sneaky big change! What made you switch to this, I'm interested in what you found?